BusBoard.ConsoleApp/Bus.cs

```diff
@@ -1,7 +1,20 @@
-﻿namespace BusBoard.ConsoleApp
+﻿using System;
+
+namespace BusBoard.ConsoleApp
 {
-    public class Bus
+    public class Bus : IComparable
     {
-        
+       public string VehicleId { get; set; }
+       public string Bearing { get; set; }
+       public DateTime ExpectedArrival { get; set; }
```

Comment:
I think you might want to use `TimeToStation` rather than `ExpectedArrival` here as it will do the time calculation for you.